minar-platform/minar_platform_types.h

```diff
@@ -27,8 +27,15 @@ namespace platform {
 enum Constants{
     // ticks per second (maximum resolution). This is what the OS works with.
     Time_Base = MINAR_PLATFORM_TIME_BASE,
+
+#if YOTTA_CFG_MINAR_TEST_CLOCK_OVERFLOW
```

Comment:
Why not define the mask itself as the value of `YOTTA_CFG_MINAR_TEST_CLOCK_OVERFLOW`? Seems like it'd give you a bit more flexibility.